sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -1988,6 +1989,24 @@ public ReadChangeStream withMetadataTableAppProfileId(String appProfileId) {
           .build();
     }
 
+    /**
+     * Returns a new {@link BigtableIO.ReadChangeStream} that overrides the config of data and/or
+     * admin client for streaming changes and for managing the metadata. For testing purposes only.
+     * Not intended for use.
+     *
+     * <p>Does not modify this object.
+     */
+    @VisibleForTesting
```

Comment:
It says visible for testing, but there are no tests. It seems like you can test this by setting up the `PipelineOptions` in the test before you expand the `BigTableChangeStreamsAccessor`

Reply:
IIUC this would require exposing all of the override parameters as PipelineOptions to users? We use this to connect to our internal testing environment. It requires fairly complicated client settings overrides that I don't think would ever be useful to expose to users

Reply:
Oh I misread the whole change and thought there were already options for this (those do tend to be useful for redirecting a whole test at a different bigtable env or local endpoint). Nevermind for now!